modules/custom/az_global_footer/az_global_footer.install

```diff
@@ -139,3 +139,34 @@ function az_global_footer_update_1020703() {
 
   return t('Updated %count menu link(s) from A-Z Index to Directory in the global footer.', ['%count' => $updated_count]);
 }
+
+/**
+ * Update Twitter link(s) to new X link(s)
+ *
+ * Twitter icon(s) in the global footer will be updated with
+ * new X icon(s).
+ */
+function az_global_footer_update_1020801() {
+  $updated_count = 0;
+  $menu_link_content_ids = \Drupal::entityQuery('menu_link_content')
+    ->accessCheck(FALSE)
+    ->condition('menu_name', 'az-footer-resources')
+    ->condition('title', 'X')
+    ->condition('link__uri', 'https://x.com/uarizona')
+    ->execute();
+
+  foreach ($menu_link_content_ids as $id) {
+    /** @var \Drupal\menu_link_content\Entity\MenuLinkContent $menu_link_content */
+    $menu_link_content = MenuLinkContent::load($id);
+    if ($menu_link_content !== NULL) {
+      // phpcs:disable
+      /* @phpstan-ignore-next-line */
```

Comment:
Investigate why these lines are needed before merging @tadean @bberndt-uaz

Reply:
The PHPStan error suppressed is:
```
Access to an undefined property Drupal\Core\Field\FieldItemList::$uri.                      
     💡 Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property
```
The PHPCS warning suppressed is: 
```
Filesystem function link used but not as a function
```

Since we have used the same code to update other Global Footer links (see the previous update functions in this file), I think we can go ahead and merge this without changing these lines.